js/private/js_binary.bzl

```diff
@@ -247,6 +247,7 @@ _ATTRS = {
         # TODO(2.0): make this mandatory so that downstream binary rules that inherit these attributes are required to set it
         mandatory = False,
     ),
+    "toolchain": attr.label(),
```

Comment:
Hmmm. Lets call this `node_toolchain` actually. Since these attributes will be inherited by downstream rules such as `jest_test`, `toolchain` on its own in that context will be confusing.